### PR TITLE
Qingwei/fix issue

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

**Comment by qingweibinary** [comment]
WIP, do not merge

File: public/styles.css [hunk 1]
```diff
@@ -1073,6 +1073,7 @@ input[type=number]::-webkit-outer-spin-button {
 .mobile-content .trade-params > div {
     margin-right: .5rem;
     min-width: 10rem;
```
Comment (nuruddeensalihu) [inline public/styles.css]: I think we shouldn't set the min-width here , It stretch the control where not required and tend to compensate with spaces where the control cannot fit the req. I mentioned it my new PR . I am of the opinion that in this case , we set for a control individually where necessary.

Comment (qingweibinary) [inline public/styles.css]: My reasoning is simple and possibly flawed, having min-width will make sure all control have similar sizes, which would looks better.

Is this reasoning sound? or is the current approach problematic?

Comment (nuruddeensalihu) [inline public/styles.css]: I was thinking about the controls variations and forgot that the screen/trade size varies also. I think you have a point considering container width changes with the layout. Controls tend to stretch sometimes when minimum width is set on the parent container. This usually is the case with radio and checkboxes who tend to compensate the minimum width with margins in most cases. I think we are fine in this case now that i check 😉

File: public/styles.css [hunk 2]
```diff
@@ -1073,6 +1073,7 @@ input[type=number]::-webkit-outer-spin-button {
...
+    max-width: 15rem;
```
Comment (borisyankov) [inline public/styles.css]: Unlikely to be a good idea, unlikely to be a terrible idea ;)

**Pull request merged**